mlrun/db/filedb.py

```diff
@@ -180,19 +184,59 @@ def del_artifacts(self, name='', project='', tag='', labels=[]):
 
                 self._safe_del(p)
 
+    def store_function(self, func, name, project='', tag=''):
+        data = self._dumps(func)
+        filepath = self._filepath(functions_dir, project, name) + self.format
+        self._datastore.put(filepath, data)
+        filepath = self._filepath(
```

Comment:
only need to write w tag (skip the first one)

Reply:
OK